--- a/common.pdesc
+++ b/common.pdesc
@@ -34,8 +34,13 @@
     isalist = "sparcv9","sparcv8plus","sparcv8","sparcv8-fsmuld","sparcv7","sparc"; \
     compatible = { "SUNW,UltraSPARC-T1", "SUNW,sun4v-cpu", "sun4v" }; \
     max-#tsb-entries = 0x2; \
-    mmu-page-size-list = 0x9; \
+    mmu-page-size-list = 0xb; \
     mmu-#va-bits = 0x40; \
+    mmu-#context-bits = 0xd; \
+    q-cpu-mondo-#bits = 0x8; \
+    q-dev-mondo-#bits = 0x8; \
+    q-resumable-#bits = 0x6; \
+    q-nonresumable-#bits = 0x2; \
     back -> cpus; \
     fwd -> cache; \
     }
@@ -81,6 +86,7 @@
      serial# = 11223344;
      stick-frequency = 5000000;
      clock-frequency = 5000000;
+     domaining-enabled = 1;
 }
 
 
